ambry-router/src/test/java/com/github/ambry/router/OperationQuotaChargerTest.java

```diff
@@ -66,7 +67,10 @@ public void testCharge() throws Exception {
     operationQuotaCharger = new OperationQuotaCharger(quotaChargeCallback, BLOBID, "GetOperation");
 
     // charge should return false if quotaChargeCallback throws exception and isCharged is false.
-    Mockito.doThrow(new RouterException("", RouterErrorCode.TooManyRequests)).when(quotaChargeCallback).charge();
+    Mockito.doThrow(
+        new QuotaException("too many requests", new RouterException("", RouterErrorCode.TooManyRequests), true))
```

Comment:
Are you planning to implement the logic to use isRetryable since the test set to true without real testing it.

Reply:
The implementation and test for isRetryable behavior will come in next PRs.